cradlepoint/pages/api/getActiveEngagements.js

```diff
@@ -6,9 +6,8 @@ export default async (req, res) => {
   // All status codes greater than 13 are for inactive engagements
   const query = { statusCode: { $lt: 13 } };
 
-  const client = await connectToDb();
-  const cursor = await client.collection("engagements").find(query);
-  
+  const db = await connectToDb();
+  const cursor = db.collection("engagements").find(query);
```

Comment:
Please don't change multiple things in one PR. Each PR should only handle one issue at a time